src/Land/Model/SoilWaterParameterizations.jl

```diff
@@ -0,0 +1,530 @@
+        vm::TemperatureDependentViscosity{FT},
+        T::FT,
+    ) where {FT}
+
+Returns the viscosity factor when we choose a TemperatureDependentViscosity.
+
+"""
+function viscosity_factor(
+    vm::TemperatureDependentViscosity{FT},
+    T::FT,
+) where {FT}
+    γ = vm.γ
+    T_ref = vm.T_ref
+    factor = FT(γ * (T - T_ref))
+    Theta = FT(exp(factor))
+    return Theta
+end
+
+
+"""
+    NoImpedance{FT} <: AbstractImpedanceFactor{FT}
+
+A model to indicate to dependence on ice for the hydraulic conductivity.
+"""
+struct NoImpedance{FT} <: AbstractImpedanceFactor{FT} end
+
+
+
+"""
+    IceImpedance{FT} <: AbstractImpedanceFactor{FT}
+
+The necessary parameters for the empirical impedance factor due to ice.
+
+# Fields
+$(DocStringExtensions.FIELDS)
+
+"""
+Base.@kwdef struct IceImpedance{FT} <: AbstractImpedanceFactor{FT}
+    "Empirical coefficient from Hansson 2014. "
+    Ω::FT = FT(7)
+end
+
+"""
+    impedance_factor(
+        imp::NoImpedance{FT},
+        θ_ice::FT,
+        porosity::FT,
+    ) where {FT}
+
+Returns the impedance factor when no effect due to ice is desired. 
+Returns 1.
+
+The other arguments are included to unify the function call.
+"""
+function impedance_factor(
+    imp::NoImpedance{FT},
+    θ_ice::FT,
+    porosity::FT,
+) where {FT}
+    gamma = FT(1.0)
+    return gamma
+end
+
+"""
+    impedance_factor(
+        imp::IceImpedance{FT},
+        θ_ice::FT,
+        porosity::FT,
+    ) where {FT}
+
+Returns the impedance factor when an effect due to the fraction of 
+ice is desired. 
+
+"""
+function impedance_factor(
+    imp::IceImpedance{FT},
+    θ_ice::FT,
+    porosity::FT,
+) where {FT}
+    Ω = imp.Ω
+    S_ice = θ_ice / porosity
+    gamma = FT(10.0^(-Ω * S_ice))
+    return gamma
+end
+
+"""
+    hydraulic_conductivity(
+        impedance::AbstractImpedanceFactor{FT},
+        viscosity::AbstractViscosityFactor{FT},
+        moisture::AbstractMoistureFactor{FT},
+        hydraulics::AbstractHydraulicsModel{FT},
+        θ_ice::FT,
+        porosity::FT,
+        T::FT,
+        S_l::FT,
+    ) where {FT}
+
+Returns the hydraulic conductivity.
+
+
+"""
+function hydraulic_conductivity(
+    impedance::AbstractImpedanceFactor{FT},
+    viscosity::AbstractViscosityFactor{FT},
+    moisture::AbstractMoistureFactor{FT},
+    hydraulics::AbstractHydraulicsModel{FT},
+    θ_ice::FT,
+    porosity::FT,
+    T::FT,
+    S_l::FT,
+) where {FT}
+    K = FT(
+        viscosity_factor(viscosity, T) *
+        impedance_factor(impedance, θ_ice, porosity) *
+        moisture_factor(moisture, hydraulics, S_l),
+    )
+    return K
+end
+
+"""
+    
+    hydraulic_head(z,ψ)
+
+Return the hydraulic head.
+
+The hydraulic head is defined as the sum of vertical height z and 
+pressure head ψ; meters.
+"""
+hydraulic_head(z, ψ) = z + ψ
+
+
+
+"""
+   effective_saturation(porosity::FT, θ_l::FT)
+
+Compute the effective saturation of soil.
+
+
+θ_l is defined to be zero or positive. If θ_l is negative, 
```

Comment:
```suggestion
`θ_l` is defined to be zero or positive. If `θ_l` is negative, 
```
Without adding back-ticks, the underscore triggers _italics_ in markdown mode (see what this looks like [here](https://clima.github.io/ClimateMachine.jl/previews/PR1279/APIs/Land/SoilWaterParameterizations/)). Entering code mode (single back-tics), or math mode (with double back-tics), seems reasonable here.

Reply:
got it! I also noticed that some of the "where"s are bolded in the lines looking like : ) where {FT}, but in the code there is no apparent difference. It seems like __word__ and **word** would bold it, but that isn't happening...